src/towncrier/newsfragments/588.bugfix

```diff
@@ -0,0 +1 @@
+Fix orphans consisting of only digits (e.g. '+12345678.feature') losing their orphan status.
```

Comment:
This message sound more like a git comment, and less like a release note.

We might need a terminology glossary  in the documentation.


I never felt the need for this feature.
I alwasy have to check, what orphan fragments are :)


 ```suggestion
Orphans fragments, fragments without an associated URL, consisting of only digits (e.g. '+12345678.feature') now retain the leading marker character.
```

Reply:
@adiroiban did you mean to use plural or singular in the first word "Orphans" in your suggested change?

Reply:
How about "non-issue fragments"?